core/essentials.go

```diff
@@ -28,7 +28,7 @@ Please refer to our documentation at https://documentation.vanillaos.org/`)
 }
 
 func CheckContainerTools() error {
-	_, distrobox := os.Stat("/usr/lib/apx/distrobox")
+	_, distrobox := os.Stat("/usr/bin/distrobox")
```

Comment:
shouldn't this point to settings.Cnf.DistroboxPath?

Reply:
fixed